lib/pages/register/register.dart

```diff
@@ -61,9 +60,9 @@ class RegisterController extends State<Register> {
   }
 
   bool _validatePassword(String password) {
-    // Define regex to validate password format
+    // Define regex to validate password format and length
     final RegExp passwordRegex =
-        RegExp(r'^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{3,16}$');
+        RegExp(r'^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,64}$');
```

Comment:
regex pattern is way too restrictive. Instead of using a regex, just check that `8 <= size <= 64`.

Further reading on password policies : https://www.ory.sh/docs/concepts/password-policy